internal/cache/file/cache_handler_test.go

```diff
@@ -479,8 +480,8 @@ func (chrT *cacheHandlerTest) Test_GetCacheHandle_ConcurrentSameFile() {
 		var err error
 		cacheHandle, err := chrT.cacheHandler.GetCacheHandle(minObj, chrT.bucket, false, 0)
 
-		AssertEq(nil, err)
-		AssertEq(nil, cacheHandle.validateCacheHandle())
+		assert.NoError(chrT.T(), err)
+		assert.Nil(chrT.T(), cacheHandle.validateCacheHandle())
```

Comment:
nit: If we receive an error, perhaps we should not validate the cacheHandle?
```suggestion
		if assert.NoError(chrT.T(), err) {
		  assert.Nil(chrT.T(), cacheHandle.validateCacheHandle())
		}
```

Reply:
Yeah, I also believe it's not required. However, I don't want to remove this as part of this PR. Will take it as separate task from pd_comp_test_* PRs.